src/com/massivecraft/factions/cmd/CmdFactionsRelationCheck.java

```diff
@@ -0,0 +1,38 @@
+package com.massivecraft.factions.cmd;
+
+import com.massivecraft.factions.Rel;
+import com.massivecraft.factions.cmd.type.TypeFaction;
+import com.massivecraft.factions.entity.Faction;
+import com.massivecraft.massivecore.MassiveException;
+
+public class CmdFactionsRelationCheck extends FactionsCommand
+{
+	// -------------------------------------------- //
+	// CONSTRUCT
+	// -------------------------------------------- //
+
+	public CmdFactionsRelationCheck()
+	{
+		// Parameter
+		this.addParameter(TypeFaction.get(), "faction");
+		this.addParameter(TypeFaction.get(), "faction", "you");
```

Comment:
Should it be of concern at all that two parameters are being given the same name?

Reply:
It is fine considering the order of the factions doesn't matter.